x-pack/plugins/observability_solution/apm/public/components/app/service_inventory/multi_signal_inventory/table/title_with_explanation.tsx

```diff
@@ -0,0 +1,162 @@
+import { i18n } from '@kbn/i18n';
+import { EuiPopover, EuiIcon, EuiText } from '@elastic/eui';
+import { useBoolean } from '@kbn/react-hooks';
+import { css } from '@emotion/react';
+
+interface Props {
+  name?: string;
+  description: ReactElement | string;
+  formula: string;
+  'data-test-subj'?: string;
+}
+
+function TitleWithPopoverExplanation(props: Props) {
+  const { name, description, formula } = props;
+  const [isPopoverOpen, { off: closePopover, toggle: togglePopover }] = useBoolean(false);
+
+  return (
+    <>
+      {name && <>{name}</>}
+      <EuiPopover
+        panelPaddingSize="s"
+        focusTrapProps={{
+          returnFocus: false,
+        }}
+        button={
+          <button
+            onClick={(e) => {
+              e.stopPropagation();
+              togglePopover();
+            }}
+            css={css`
+              display: flex;
+            `}
+            data-test-subj={props['data-test-subj']}
+          >
+            <EuiIcon type="questionInCircle" />
+          </button>
+        }
+        isOpen={isPopoverOpen}
+        offset={10}
+        closePopover={closePopover}
+        repositionOnScroll
+        anchorPosition="upCenter"
+        panelStyle={{ maxWidth: 350 }}
+      >
+        <EuiText size="xs">
+          <p>{description}</p>
+          <p>
+            <strong>
+              <FormattedMessage
+                id="xpack.apm.multiSignal.tooltip.formula"
+                defaultMessage="Formula Calculation:"
+              />
+            </strong>
+            <br />
+            <code
+              css={css`
+                word-break: break-word;
+              `}
+            >
+              {formula}
+            </code>
+          </p>
+        </EuiText>
+      </EuiPopover>
+    </>
+  );
+}
+
+interface ExplanationProps {
+  includeTitle: boolean;
+}
+
+export function LogRateWithExplanation({ includeTitle }: ExplanationProps) {
+  return (
+    <TitleWithPopoverExplanation
+      name={
+        includeTitle
+          ? i18n.translate('xpack.apm.multiSignal.servicesTable.logRatePerMinute', {
+              defaultMessage: 'Log rate (per min.)',
+            })
+          : undefined
+      }
+      description={
+        <FormattedMessage
+          defaultMessage="Rate of logs per minute observed for given {serviceName}."
+          id="xpack.apm.multiSignal.servicesTable.logRatePerMinute.tooltip.description"
+          values={{
+            serviceName: (
+              <code
+                css={css`
+                  word-break: break-word;
+                `}
+              >
+                {i18n.translate(
+                  'xpack.apm.multiSignal.servicesTable.logRatePerMinute.tooltip.serviceNameLabel',
+                  {
+                    defaultMessage: 'service.name',
+                  }
+                )}
+              </code>
+            ),
+          }}
+        />
+      }
+      formula="count(kql='log.level: * OR error.log.level: *')"
```

Comment:
Where can I find this formula? Is it in the transform?

Reply:
I tried this one but I don't think that's what we are showing there:
<img width="1834" alt="image" src="https://github.com/elastic/kibana/assets/14139027/9182ebc6-c4df-410f-a98f-1e2b98f3f8c3">
so what we do there is filter by `log.level` and show it in the table and the chart (as a percentage value for a period of time)
Do you know if that's the correct formula for that 🤔 Where did you find the `counter_rate(count())` or `differences(cumulative_sum(count()))`? I am talking about the formula for the chart and the table here: 
<img width="809" alt="image" src="https://github.com/elastic/kibana/assets/14139027/625bc34e-3e19-4b37-8657-856dc314a569">

@kpatticha Can you please help us here? Do you think we should show those formulas (`counter_rate(count())` or `differences(cumulative_sum(count()))` ) instead? I am trying to make sure that we have the correct information displayed based on the queries you did to get the data

Reply:
I think you need to point to the `logs` data-data view?

Reply:
Infra is different as we use Lens there and imo it's more straight. In APM, we build these charts. 

>  Where did you find the counter_rate(count()) or differences(cumulative_sum(count()))

I'm also super familiar with those. Can you please explain? 

> Where can I find this formula? Is it in the transform? 

For the table, yes https://github.com/elastic/kibana/pull/187021/files#diff-949da5330a90347982e4aae029dcdb7c9d5c43e8fb59263a222f2ccd541a2f99R106

Reply:
@kpatticha `counter_rate(count())` or `differences(cumulative_sum(count()))` is how you would represent a rate calculation using tiny math, which is what lens uses. If users go into lens and try to build a chart using the information provided in the tooltip, the chart would probably be wrong, that's what I'm concerned about.

`count` will only return the number of logs in a given time range. If we expect to represent rate, it has to be the count divided by the length of the date of the interval.